src/mapeo-project.js

```diff
@@ -345,6 +346,20 @@ export class MapeoProject {
     await Promise.all([this.#coreManager.ready(), this.#ownershipWriteDone])
   }
 
+  /**
+   */
+  async close() {
+    this.#l.log('closing project %h', this.#projectId)
+    this.#coreManager.close()
+
+    // eslint-disable-next-line no-unused-vars
+    for await (let [_, dataStore] of Object.entries(this.#dataStores)) {
+      await dataStore.close()
+    }
```

Comment:
This is not an async iterator, so no need for `for await`. Also just use `Object.values()`. Also it's not performance critical, but better to close in parallel rather than serial.

```suggestion
    for (const dataStore of Object.values(this.#dataStores)) {
      await dataStore.close()
    }
```